src/toolchain/toolchain.ts

```diff
@@ -346,6 +346,9 @@ export class SwiftToolchain {
      */
     public async getLLDB(): Promise<string> {
         let lldbPath = path.join(this.swiftFolderPath, "lldb");
```

Comment:
How about sinking this into an `else` condition after the `win32` check? Alternatively, you could write this as a ternary:

```javascript
  let lldbPath = path.join(this.swiftFolderPath, process.platform === 'win32' ? 'lldb.exe' : 'lldb')
```

Reply:
That works, too. Done!